### PR TITLE
fix: Add img alt

### LINKED ISSUE #308: Some Images missing "alt" property
## Bug
Some images are missing alt properties and have the following error message on the terminal when it's run locally. 

"Image is missing required "alt" property. Please add Alternative Text to describe the image for screen readers and search engines."

## Fix 
Make sure to give alt properties to all the images on the page and check if the message has gone away.

### Screenshots

Screenshot from 2023-11-05 08-16-36
Screenshot from 2023-11-05 08-18-02

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lexzee**
## Fixes Issue 
Closes #308

## Changes proposed

- [x] Make sure to give alt properties to all the images on the page and check if the message has gone away.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/components/Newsletter.jsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ export function Newsletter() {
             Awards
           </h3>
           <div className="my-4 flex flex-wrap items-center justify-evenly gap-4">
-            <Image src={CommunityAward} width="500" className="rounded" />
-            <Image src={TeacherAward} width="500" className="rounded" />
+            <Image alt={"Community Award"} src={CommunityAward} width="500" className="rounded" />
```
Comment (xkrishguptaa) [inline src/components/Newsletter.jsx:34]: Can we have a better alt text? For example: "Image Showcasing the GitHub Stars Nova 2021 Award, awarded to Eddie Jaoude, Founder of EddieHub"

File: src/components/PrimaryFeatures.jsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ export function PrimaryFeatures() {
                       <Image
                         className="w-full"
                         src={feature.image}
-                        alt=""
+                        alt={feature.title}
```
Comment (xkrishguptaa) [inline src/components/PrimaryFeatures.jsx:132]: wouldn't this be repetitive as the screen reader would first read this and then read the title which is a header anyways

Comment (lexzee) [reply src/components/PrimaryFeatures.jsx:132]: I will check it out and make corrections

File: src/components/Newsletter.jsx (hunk 1 above)
Comment (lexzee) [reply src/components/Newsletter.jsx:34]: Thanks for the suggestion, I'll update the code

**Comment by xkrishguptaa** [comment]
Hey @lexzee, could you please fix the merge conflicts so that we can merge your PR :)

**Review by lexzee — COMMENTED** [review]
I can't find any bug, do you mind pointing out what I missed

**Comment by xkrishguptaa** [comment]
Hi! It is not an error but a merge conflict, it is when two different versions of Git Files have to be merged together: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

If you need any help, let us know and we'd be happy to help!

**Comment by lexzee** [comment]
I have resolved the conflict

**Comment by xkrishguptaa** [comment]
@lexzee have you pushed your changes after resolving it? It still shows conflicts over here

**Comment by lexzee** [comment]
@xkrishguptaa the conflict should have been resolved now

**Review by xkrishguptaa — APPROVED** [review]

**Comment by xkrishguptaa** [comment]
Thank you for your contribution!

**Pull request merged**